examples/page-redder/background.js

```diff
@@ -3,8 +3,10 @@ function reddenPage() {
 }
 
 chrome.action.onClicked.addListener((tab) => {
-  chrome.scripting.executeScript({
-    target: { tabId: tab.id },
-    function: reddenPage
-  });
+  if(!tab.url.includes("chrome://")) {
```

Comment:
I _think_ this should only match against the URL's scheme, but I think it would be a bit safer to replace `includes()` with `startsWith()` in order to avoid situations where this string appears in other parts of the URL. 

```suggestion
  if(!tab.url.startsWith("chrome://")) {
```